lib/tasks/report.rake

```diff
@@ -61,9 +61,9 @@ namespace :report do
     Reports::DataExporter.new.export_csv_from_slugs_at(args.date, args.extras)
   end
 
-  desc "Export the number of subscriptions for the 'Living in' taxons for European countries"
-  task csv_from_living_in_europe: :environment do
-    Reports::DataExporter.new.export_csv_from_living_in_europe
+  desc "Export the number of subscriptions for the 'Living in' taxons for European countries as of a given date (format: 'yyyy-mm-dd')"
+  task :csv_from_living_in_europe, [:date] => :environment do |_, args|
```

Comment:
Can we update [the test](https://github.com/alphagov/email-alert-api/blob/master/spec/lib/tasks/report_spec.rb#L84) as well? Kind of surprised it's passing 🤔